### PR TITLE
Explain new Controller hooks

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Explain new Controller hooks
| Fixed ticket? | Fixes https://github.com/PrestaShop/PrestaShop/issues/21379

### CODE REVIEW

File: src/content/1.7/modules/core-updates/1.7.7.md [hunk 1]
```diff
@@ -95,6 +95,24 @@ The following hooks have been added:
 - displayAdditionalCustomerAddressFields
 - displayFooterCategory
 - actionAdminAdminPreferencesControllerPostProcessBefore
+- actionAdminLoginControllerBefore
+- actionAdminLoginControllerLoginBefore
+- actionAdminLoginControllerLoginAfter
+- actionAdminLoginControllerForgotBefore
+- actionAdminLoginControllerForgotAfter
+- actionAdminLoginControllerResetBefore
```
Comment (okom3pom) [inline src/content/1.7/modules/core-updates/1.7.7.md:103]: actionAdminLoginControllerResetAfter

Comment (okom3pom) [reply src/content/1.7/modules/core-updates/1.7.7.md:103]: It's my first time i add a comment i don't know if it's good. :baby:

Comment (PierreRambaud) [inline src/content/1.7/modules/core-updates/1.7.7.md:103]:
```suggestion
- actionAdminLoginControllerResetAfter
```

File: src/content/1.7/modules/core-updates/1.7.7.md [hunk 2]
```diff
@@ -95,6 +95,24 @@ The following hooks have been added:
 - displayAdditionalCustomerAddressFields
 - displayFooterCategory
 - actionAdminAdminPreferencesControllerPostProcessBefore
+- actionAdminLoginControllerBefore
+- actionAdminLoginControllerLoginBefore
+- actionAdminLoginControllerLoginAfter
+- actionAdminLoginControllerForgotBefore
+- actionAdminLoginControllerForgotAfter
+- actionAdminLoginControllerResetAfter
```
Comment (PierreRambaud) [inline src/content/1.7/modules/core-updates/1.7.7.md:103]:
```suggestion
- actionAdminLoginControllerResetBefore
- actionAdminLoginControllerResetAfter
```

**Review by PierreRambaud — APPROVED** [review]

**Review by okom3pom — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @matks

**Pull request merged**